sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormFieldType.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Azure.Core;
+
+namespace Azure.AI.FormRecognizer.Models
+{
+    /// <summary>
+    /// </summary>
+    [CodeGenModel("FieldValueType")]
+    public enum FieldValueType
```

Comment:
Class name and enum don't match. Is that intended?

Reply:
You mean the file name?  Our static analyzer rules don't pick up mismatched filenames for enums, so I missed this.  The file name should be FormFieldType.cs, thanks for catching that!

Reply:
Yes! sorry, file name :|.
Just to clarify, the file name should be `FieldValueType` instead of `FormFieldType` no?

Reply:
Yes, it should be FieldValueType.cs.  The name of the enum here is correct.